src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -812,6 +843,40 @@ public static void ActionDoesNotExplicitlySpecifyContentTypes(this ILogger logge
             }
         }
 
+        public static void MultipleRangesNotSupported(this ILogger logger)
```

Comment:
Don't see this being used.

Reply:
I didn't realize that the class where I used this is actually part of a 'sources' file. This sources file (RangeHelper.cs) is in StaticFiles and intend to make that change in that repo. Will send out a PR for it.